src/webapp/TourneyRent/src/app/resources/team.resource.ts

```diff
@@ -13,11 +13,23 @@ export class TeamResource {
     return this.httpClient.post(`${API_URL}/Team`, data);
   }
 
+  public getAllTeams(): Observable<any> {
+    return this.httpClient.get(`${API_URL}/Team`);
+  }
+
+  public getTeamMembers(teamId: any): Observable<any> {
+    return this.httpClient.get(`${API_URL}/Team/${teamId}/members`);
+  }
+
   public getUserTeams(userId: any): Observable<any> {
     return this.httpClient.get(`${API_URL}/Team/Members/${userId}`);
   }
 
   public addTeamMember(teamId: any, teamMemberCreate: any): Observable<any> {
     return this.httpClient.post(`${API_URL}/Team/${teamId}/members`, teamMemberCreate);
   }
+
+  public getProfile(userId: any): Observable<any> {
+    return this.httpClient.get(`${API_URL}/Account/${userId}`);
+  }
 }
```

Comment:
I think profile.resource.ts has this method, so you can just inject ProfileResource where you need this